tests/rptest/test_suite_scale.yml

```diff
@@ -0,0 +1,7 @@
+ec2:
+  included:
+    #
+    # Feel free to add any large scale tests you run separately here.
```

Comment:
How long does it take?  "nightly" really just means once every 24 hours, so we have a fairly large time budget for nightly runs -- a test that takes e.g. 1hr is totally fine.

Reply:
Depends on parameters, but let's say four hours and a lot of S3 queries (which costs $$).

Reply:
I'm a bit surprised: this runtime could be pointing to a real issue.  By rough calculation, the test is only writing about 12GB of data: it shouldn't be taking hours to run, even on smaller i3en.xlarge nodes.  Do you have a sense of where that runtime is coming from?

Reply:
Yes there was a real bug causing the long runtime for that small amount of data: #5928. The existing shorter tests did not catch this bug. The intention, though, is for this to be a long-running, somewhat expensive test, that runs per-release or maybe weekly (when we have that).

Reply:
Also.. there are a lot of manifests and objects in general, since there are partitions in the thousands. Even with the increased number of uploaders, it is pretty slow. I will be posting an update shortly that increases the scale now that we have a fix for #5928

Reply:
Good that this caught a real bug: would be very interesting to see the new runtime after #5934 is fixed.

Given that we don't have weekly runs yet, let's look for a way to get this hooked in so that it doesn't bitrot between releases: if the runtime is <1hr then I think it makes sense to include it in the existing nightly runs: would rather keep these tests fresh and move them out to weekly as/when, rather than leaving them in the tree but not hooked in to regular runs.

Reply:
Runtime is still really long. Good points though. 

 Most recent run attempted to increase scale and reduce "wait for upload to cloud" time, but still timed out after 4 hours:
```
---------------------------------------------------------------------------------------------------------------------------------------------------------------
test_id:    rptest.scale_tests.extreme_recovery_test.ExtremeRecoveryTest.test_recovery_scale
status:     FAIL
run time:   243 minutes 49.978 seconds


    TimeoutError('objects not found in S3')
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/ducktape/tests/runner_client.py", line 135, in run
    data = self.run_test()
  File "/home/ubuntu/.local/lib/python3.10/site-packages/ducktape/tests/runner_client.py", line 227, in run_test
    return self.test_context.function(self.test)
  File "/home/ubuntu/redpanda/tests/rptest/services/cluster.py", line 35, in wrapped
    r = f(self, *args, **kwargs)
  File "/home/ubuntu/redpanda/tests/rptest/scale_tests/extreme_recovery_test.py", line 179, in test_recovery_scale
    self.do_run(extremeRecovery, RecoveryScale.expected_transfer_sec)
  File "/home/ubuntu/redpanda/tests/rptest/tests/topic_recovery_test.py", line 1094, in do_run
    self._wait_for_data_in_s3(
  File "/home/ubuntu/redpanda/tests/rptest/tests/topic_recovery_test.py", line 1035, in _wait_for_data_in_s3
    wait_until(verify,
  File "/home/ubuntu/.local/lib/python3.10/site-packages/ducktape/utils/util.py", line 58, in wait_until
    raise TimeoutError(err_msg() if callable(err_msg) else err_msg) from last_exception
ducktape.errors.TimeoutError: objects not found in S3

---------------------------------------------------------------------------------------------------------------------------------------------------------------
===============================================================================================================================================================
SESSION REPORT (ALL TESTS)
ducktape version: 0.8.8
session_id:       2022-08-11--006
run time:         243 minutes 50.066 seconds
tests run:        8
passed:           0
failed:           1
ignored:          7
opassed:          0
ofailed:          0
===============================================================================================================================================================
```
I'll keep tweaking and testing to see if I can come up with a "happy medium" that can fit within nightlies--until we get a longer schedule set up.

On the S3 cost--we probably need to measure, but I've seen metadata requests add up to significant $$ in the past, and this test does a ton of them polling for the thousands of objects to arrive in s3. That could be optimized a little, but I'm still in the habit of avoiding these costs where possible (i.e. by not running it nightly).

Reply:
Test has been pared down and optimized, and is now included in CDT nightlies, resolving.